rc-events-service/src/main/scala/com/epam/random_coffee/events/model/Event.scala

```diff
@@ -0,0 +1,16 @@
+package com.epam.random_coffee.events.model
+
+import akka.http.scaladsl.server.{ PathMatcher, PathMatcher1 }
+import akka.http.scaladsl.server.PathMatchers.Segment
+
+final case class Event(id: EventId, eventName: String)
+
+final case class EventId(value: String) extends AnyVal
+
+object EventId {
+
+  def stringToOptionId(from: String): Option[EventId] = Some(EventId(from))
+
+  val eventIdMatcher: PathMatcher1[EventId] =
+    PathMatcher(Segment).flatMap(path => EventId.stringToOptionId(path))
```

Comment:
1) Why would `EventId` depend on Akka http? Remember I told you that repo layer doesn't depend on service layer and service layer doesn't depend on API layer? Model layer is even lower in this hierarchy and cannot depend on things from repo layer, let alone service and API layers.
2) What's this strange `EventId.stringToOptionId` method do? Can't you just do EventId(_) ?

Reply:
I moved the matcher to API layer and rewrote this line